src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/ConfigurationBinder/Bind_NamedParameters.generated.txt

```diff
@@ -0,0 +1,170 @@
+// <auto-generated/>
```

Comment:
I think these would be better as regular runtime tests. I'd put them here instead https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/ConfigurationBinderTests.Generator.cs.

For the `OptionsBuilder` and `IServiceCollection` extension methods, I'd add tests to https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Options.ConfigurationExtensions/tests/SourceGenerationTests/ConfigurationExtensionsTest.Generator.cs.

Like others in the files I shared, specific syntax patterns are crucial for these new tests. The comments need to be clear on what scenario is being tested so that future devs won't change them.

Reply:
> I think these would be better as regular runtime tests. I'd put them here instead https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/ConfigurationBinderTests.Generator.cs.

I did added test there, for repro removed the fix and run tests but test did not fail, for example I added this::

```cs
[Fact]
public void NamedParameterTests_Bind()
{

    IConfiguration configuration = TestHelpers.GetConfigurationFromJsonString("""{"MyString":"hi", "MyList":[1], "MyInt":1}""");

    var m1 = new MyClass();

    ConfigurationBinder.Bind(instance: m1, configuration: configuration);

    Assert.Equal(1, m1.MyInt);
    Assert.Equal("hi", m1.MyString);
    Assert.Equal(1, m1.MyList[0]);
}

public class MyClass
{
    public string MyString { get; set; }
    public int MyInt { get; set; }
    public List<int> MyList { get; set; }
    public Dictionary<string, string> MyDictionary { get; set; }
}
```

I could not debug because my VS showing error, so could not check which Bind method it is calling (generated or not), I guess the test doesn't throw/fail when the generator did not produce source and it silently calls the original Bind method.

Reply:
Or am I missing something here @layomia ?

Reply:
Confirmed offline that the test falling back to use reflection version. As discussed with @layomia removed the text files added, and just testing that the file is generated and row count, no need to test row by row as other tests already doing that